### DISCUSSION TITLE
ReadMe Update: Worlds update appears to have interfered with some of the Expedition 5 planets

### DISCUSSION THREAD

**Discussion by eSonOfAnder** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#24] [Ideas]
Might be worth adding a note in the main ReadMe file of the project, and/or in the specific ReadMe for Expedition 5 that the Worlds update has changed some of the world generation for the expedition planets. Specifically, for the Eggs milestone, the expedition states that the Rendezvous 3 planet contains the creatures needed for Giant Eggs (diplos), but that is no longer the case. I am currently scouring the neighboring systems to the Expedition systems to locate any of the egg producing creatures, and have come up with nil so far. Zero striders, triceratops, or diplos to be found in the current world generation.

**Comment by ghost** [comment]
I found that out too. Did you find a creature that scan is 200,000 worth?

**Reply by cwmonkey** [reply]
Just get lots of scanner upgrades and make sure they have +units for scanning creatures.

**Comment by OwlsAreBestBirbs** [comment]
Yeah, noticed that the worlds are completely different from the guides

**Comment by bedev0001** [comment]
Yeah same with me, I just visited 300 planets to find some, but I can not show the location, because I went far off chart from the rendezvous points. 

But if expedition 5 gives you the portalglyphes, you can visit these diplos from my screenshot. 
I checked for Expedition 11 some old coords out, to find tall creatures and this one is still valid. I made this screenshot two days ago.
![20241017000749_1](https://github.com/user-attachments/assets/9e218623-5478-49a5-91c4-5e3d4643c8fc)

**Comment by Gordx** [comment]
Someone can help me i looking for tall eggs in this moment, but the guides on yt are old and differents. :(
